resolver/resolver.go

```diff
@@ -0,0 +1,11 @@
+package resolver
+
+import (
+	"github.com/qri-io/qri/dsref"
+)
+
```

Comment:
I'd appreciate a base-level "name resolution error" defined in this package.

Reply:
Added, and in use.